sampleapp/src/main/res/xml/network_security_config.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <network-security-config>
     <domain-config cleartextTrafficPermitted="true">
-        <domain includeSubdomains="true">192.168.68.103</domain>
+        <domain includeSubdomains="true">192.168.68.114</domain>
```

Comment:
Would propose to set

```
<domain includeSubdomains="true">192.168.*.*</domain> 
```

Using wildcard instead which basically will allow any of your local IP addresses and most of us will use the same ranges by default so this covers most cases and u don't need to change everytime your DHCP assigns you an IP

Reply:
I tried that and does not work unfortunately.